Gruntfile.js

```diff
@@ -74,5 +88,12 @@ module.exports = function(grunt) {
   }
 
   grunt.registerTask("dev", ["connect", "watch"]);
-  grunt.registerTask("test", ["connect", "saucelabs-custom"]);
+  grunt.registerTask("test-locally", ["connect", "jasmine"]);
+
+  var secureEnvVars = process.env.TRAVIS_SECURE_ENV_VARS;
+  if (secureEnvVars === undefined || secureEnvVars === "true") {
```

Comment:
Yes. another way to detect it would be to check if `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` are set. My guess would be that Travis doesn't set them at all if `secureEnvVars === false`. 

This way, SauceLabs would be the default for everyone with valid SauceLabs credentials.